src/Maestro/tests/Scenarios/channels.ps1

```diff
@@ -34,7 +34,7 @@ try {
     }
 
     Write-Host "Removing '$newChannelName'"
-    Darc-Command delete-channel --name `'$newChannelName`'
+    Darc-Command delete-channel --name $newChannelName
```

Comment:
Don't you need a try+catch here like in the other case?

Reply:
Perhaps although I'm not as familiar with this. @mmitche ?

Reply:
@mmitche is not back from holiday for a bit...

Reply:
It's not necessary. The first time we delete we kind of expect it to fail because there shouldn't be any channels with the random name that we assign during the tests, so that's why we ignore any errors during deletion with the try-catch.

For cleanup of the tests, we really want to make sure not to skip over failures, and want the test to fail if unable to clean up so we don't encourage resource leakage.